tests/integration/utils/test_pdf_utils.py

```diff
@@ -0,0 +1,22 @@
+import pathlib
+
+import pytest
+
+from askui.utils.pdf_utils import load_pdf
+
+
+class TestLoadPdf:
+    def test_load_pdf_from_path(self, path_fixtures_dummy_pdf: pathlib.Path) -> None:
```

Comment:
Nice tests, data url is missing :) 

See https://gist.github.com/brianguertin/b3ab21de49ac5af81aef58997adc4fac